src/node/utils/fsUtils.ts

```diff
@@ -143,3 +147,13 @@ export function watchFileIfOutOfRoot(
     watcher.add(file)
   }
 }
+
+function fileExistsWithCaseSync(filepath: string): boolean {
+  const dir = path.dirname(filepath)
```

Comment:
This seems pretty expensive to do on every initial file read. Is there any cheaper way to perform this check?

Reply:
I'm not found other vaild way to resolve this. Maybe we can provider a config to turn on this?